data/abilities/discovery/b93bd80e-3a70-11eb-adc1-0242ac120002.yml

```diff
@@ -21,10 +21,16 @@
           ./bacwi
         payloads:
           - bacwi
+        parsers:
+          plugins.bacnet.app.parsers.data_set:
```

Comment:
With the new parser name, this line needs to be updated to `plugins.bacnet.app.parsers.device_instance`